crates/katana/executor/src/implementation/blockifier/mod.rs

```diff
@@ -111,8 +105,8 @@ impl<'a> StarknetVMProcessor<'a> {
         // TODO: @kariy, not sure here if we should add some functions to alter it
         // instead of cloning. Or did I miss a function?
         // https://github.com/starkware-libs/blockifier/blob/a6200402ab635d8a8e175f7f135be5914c960007/crates/blockifier/src/context.rs#L23
```

Comment:
Looks like they hear you: https://github.com/starkware-libs/blockifier/blob/99ddb927994be44ef333c860d33ecc149e8d4273/crates/blockifier/src/context.rs#L28.